### ISSUE TITLE
spreadsheet_handle.py still in examples/

### ISSUE THREAD

**Issue by j4321** [slightlynybbled/tk_tools#41]
The example spreadsheet_handle.py has not been removed from the examples after `SpreadSheetReader` was removed from the package. So when trying to run it we get the error

```
Traceback (most recent call last):
  File "spreadsheet_handle.py", line 9, in <module>
    sheet_handle = tk_tools.SpreadSheetReader(root,
AttributeError: module 'tk_tools' has no attribute 'SpreadSheetReader'
```

**Comment by slightlynybbled** [comment]
Thank you!  I totally missed that.

**Comment by slightlynybbled** [comment]
Addressed by bc2c27b
